test/integrate/dubbo/go-client/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/apache/dubbo-go/test/integrate/dubbo/go-client
 go 1.13
 
 require (
-	github.com/apache/dubbo-go v1.5.6-rc2.0.20210405033601-abbd985fb785
+	github.com/apache/dubbo-go v1.5.5
```

Comment:
why not using 1.5.6?

Reply:
useless version number
it will be replaced when run by  this Docker file
https://github.com/apache/dubbo-go/blob/00dbac59da835dc303089bac3a82448b81fe9078/test/integrate/dubbo/go-server/Dockerfile#L31

Reply:
if so, @LaurenceLiZhixin pls add @zouyx 's comment as remark on this line.

Reply:
done